metadata_service/config.py

```diff
@@ -21,10 +21,8 @@
 from ghga_service_chassis_lib.config import config_from_yaml
 from ghga_service_chassis_lib.pubsub import PubSubConfigBase
 
-from .models import SupportedLanguages
 
-
-@config_from_yaml(prefix="my-microservice")
+@config_from_yaml(prefix="metadata-service")
```

Comment:
Sorry, this is actually wrong in the template repo. It should be snake case:
```suggestion
@config_from_yaml(prefix="metadata_repository_service")
```